1.0-E2E-status.md

```diff
@@ -132,7 +132,7 @@ Legend:
 | ToDo App                     |              |           |                          |         |          |             |
 |                              | C#           | Dapeng    | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/dotnet) |             |
 |                              | JS           | Qiaoqiao  | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/javascript)        |             |
-|                              | Java         | Weidong   | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/java) |            |
+|                              | Java         | Weidong   | :running_man:            | 1/15    | [Code](https://github.com/allenjzhang/typespec-e2e-demo/tree/main/todoApp/clients/java) | Pagination depends on clientcore beta.2. ETA 1/20. |
```

Comment:
Could you update directly in the ETA column?

Reply:
Thanks. Updated to ETA column.